CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 ## 10.4.0 (YYYY-MM-DD)
+### Breaking Changes
+* Queries no longer check do nullability checks on non-nullable fields.
 
 ### Enhancements
 * Added support for `java.util.UUID` as supported field in model classes.
 * Added support for `java.util.UUID` as a primary key.
 * Added support for the string-based Realm Query Language through `RealmQuery.rawPredicate(...)`. This allows many new type of queries not previously supported by the typed query API. See the Javadoc on this method for further details. (Issue [#6116](https://github.com/realm/realm-java/pull/6116))
-
+ 
 ### Fixes
-* None
+* Checks for null Strings on non-nullable query operators `contains`, `beginsWith`, `endsWith`, and `like`.
+
+### Known Bugs
```

Comment:
There are quite a few other known bugs in realm-java... Not sure it makes sense to list all those :-)
If both these were features that no longer works, perhaps we should list it under Breaking changes instead, and say these are known "breaking limitations"?